pkg/operator/staticpod/controller/installer/manifests/installer-pod.yaml

```diff
@@ -27,6 +27,9 @@ spec:
         limits:
           memory: 100M
   restartPolicy: Never
+  priorityClassName: system-node-critical
+  tolerations:
+    - operator: "Exists"
```

Comment:
is this correct? Does it apply to any taint?

Reply:
@sttts this is the toleration our operators are using (copied from kube-apiserver-operator).

Reply:
Didn't we have many more tolerations there?

Reply:
https://github.com/openshift/cluster-kube-apiserver-operator/commit/e7af9bd097321b1f70368568000e8e44b43adb30#diff-08a6124d2d9be934074b6601cf61dac5

I thought so as well, but no, we only added this one.